### DISCUSSION TITLE
Fonts displaying on debug mode but changes to static on production build

### DISCUSSION THREAD

**Discussion by ameysunu** [facebook/docusaurus#4107] [General]
Hello! I have been using ``@font-face`` to change fonts on the entire root which works perfectly while on ``npm start`` but however switches to static Times New Roman on the production build. I also tried using Google Fonts for the same but however this doesn't work/display on iOS devices, switches to the static Times New Roman. Can somebody help me?

**Comment by slorber** [comment]
Please use https://new.docusaurus.io to submit a repro with your failing attempt
